src/System.IO.Ports/tests/SerialPort/AbortOnError.cs

```diff
@@ -28,7 +28,7 @@ public class AbortOnError : PortsTest
         /// 3. Verify that the fAbortOnError flag is clear
         /// 
         /// </summary>
-        [ConditionalFact(nameof(HasOneSerialPort))]
+        [ConditionalFact(nameof(HasOneSerialPort), nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotUap))]
```

Comment:
Per offline discussion documenting in the code but not taking the rd.xml path (at least for now, but likely going to direct Pinvoke calls later).